config.go

```diff
@@ -535,6 +536,24 @@ func getConfig() *types.Configuration {
 		}
 	}
 
+	if value, present := os.LookupEnv("ALERTMANAGER_CUSTOM_SEVERITYMAP"); present {
+		severitymap := strings.Split(value, ",")
+		for _, severitymatch := range severitymap {
+			values := strings.SplitN(severitymatch, ":", 2)
+			priorityString := strings.TrimSpace(values[0])
+			if types.Priority(priorityString) == types.Default {
+				log.Printf("[ERROR] : AlertManager - Priority '%v' is not a valid falco priority level", priorityString)
+				continue
+			}
```

Comment:
This check should not be made at this level, but in the main handler.

Reply:
Why is that ?

Reply:
We should check in this method https://github.com/falcosecurity/falcosidekick/blob/58ec86e17441e2d39473e1727c210031b4512069/handlers.go#L78 if the Priority is valid or not, not just for AlertManager.

Reply:
This priority check only append at the startup of the sidekick as it's for setting up the custom severity map. It has nothing to do with the Payload creation and check